src/hardware_trigger.hpp

```diff
@@ -85,18 +113,28 @@ class HardwareTrigger
 	void setStreamingFlag(bool);
 	bool getStreamingFlag();
 
+	bool hasExternalTriggerIn();
```

Comment:
The `bool has*` methods could be `const`.

Reply:
fixed